src/memex/views.py

```diff
@@ -82,7 +99,41 @@ def api_config(**settings):
         request_method = ('DELETE', 'GET', 'HEAD', 'POST', 'PUT',)
     settings['request_method'] = request_method + ('OPTIONS',)
 
-    return view_config(**settings)
+    if link_name:
+        link = {'name': link_name,
+                'method': initial_request_method,
```

Comment:
Or I'm wondering if with the current set up (and taking the PUT/PATCH migration into account), it might be easiest to just take the first one? It might never practically happen that one API view responds to two different methods doing different things without them being two separate views.